src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx

```diff
@@ -208,22 +219,49 @@ export const CoverVideoPublishingStateOverlay = styled.div`
   padding: ${sizes(1)} ${sizes(2)};
   display: flex;
   align-items: center;
-  background-color: ${colors.gray['900']};
-  opacity: 0.7;
+  background-color: ${colors.black};
+  color: ${colors.white};
+  z-index: ${zIndex.overlay};
 `
 
 export const PublishingStateText = styled(Text)`
   margin-left: ${sizes(1.5)};
 `
 
+export const KebabMenuIconContainer = styled.div<{ isActive?: boolean }>`
+  display: ${({ isActive }) => (isActive ? 'flex' : 'none')};
+  align-items: center;
+  justify-content: center;
+  width: 36px;
+  height: 36px;
+  cursor: pointer;
+  position: relative;
+  border-radius: 100%;
+  transition: all ${transitions.timings.regular} ${transitions.easing};
+  margin-left: auto;
+
+  path {
+    transition: all ${transitions.timings.regular} ${transitions.easing};
+  }
+
+  &:hover {
+    path:not([fill='none']) {
+      fill: ${colors.white};
+    }
+
+    background-color: ${transparentize(1 - 0.06, colors.white)};
```

Comment:
I think we could use here `colors.transparentWhite[6]`. Also probably not your choice, but this particular color used in this context is barely visible. Do we plan changing background around video tiles in the future? Personally I don't like it,  maybe we could change it to more brighter variant?

Reply:
Good catch. After investigation, it turned out, there was a change in background color.